From 7e60aa8be1492ad682cd7698d7b0ca3c477708fb Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Tue, 31 Jul 2018 14:39:14 -0700
Subject: [PATCH] TRITON-646 the update of workflow to sdc-clients@11.x breaks
 VMAPI workflows that call SAPI Reviewed by: Todd Whiteman
 <todd.whiteman@joyent.com> Approved by: Todd Whiteman
 <todd.whiteman@joyent.com>

---
 lib/workflows/add-nics.js      | 4 ++--
 lib/workflows/fabric-common.js | 3 ++-
 lib/workflows/provision.js     | 4 ++--
 3 files changed, 6 insertions(+), 5 deletions(-)

diff --git a/lib/workflows/add-nics.js b/lib/workflows/add-nics.js
index 3254120..cabb5c1 100644
--- a/lib/workflows/add-nics.js
+++ b/lib/workflows/add-nics.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright 2017, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 /*
@@ -23,7 +23,7 @@ var async;  // stub to keep jsl happy
 var common = require('./job-common');
 var fabricCommon = require('./fabric-common');
 
-var VERSION = '7.2.0';
+var VERSION = '7.2.1';
 
 
 /*
diff --git a/lib/workflows/fabric-common.js b/lib/workflows/fabric-common.js
index 71a304e..5d03c78 100644
--- a/lib/workflows/fabric-common.js
+++ b/lib/workflows/fabric-common.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright 2017, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 /*
@@ -164,6 +164,7 @@ function provisionFabricNats(job, cb) {
     var sapi = new sdcClients.SAPI({
         log: job.log.child({ component: 'sapi' }),
         url: sapiUrl,
+        version: '~2',
         headers: { 'x-request-id': job.params['x-request-id'] }
     });
 
diff --git a/lib/workflows/provision.js b/lib/workflows/provision.js
index 0818e0f..c7a8267 100644
--- a/lib/workflows/provision.js
+++ b/lib/workflows/provision.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright 2018, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 /*
@@ -23,7 +23,7 @@ var nfsVolumes = require('./nfs-volumes');
 
 var wfapiUrl;
 
-var VERSION = '8.0.0';
+var VERSION = '8.0.1';
 
 
 /*
-- 
2.21.0

